when-shit-hits-the-fan.adoc

```diff
@@ -381,29 +381,28 @@ everyone is downloading the whole block index at once.
 I'll build releases shortly.
 ____
 
-He wanted people to download block data from a specific user,
+He wanted people to download block data from a specific user, namely
 knightmb, who had published his blockchain as it appeared on his disk,
 the files blkXXXX.dat and blkindex.dat. The reason for downloading the
 blockchain data this way, as opposed to synchronizing from scratch, was
-said to reduce network bandwidth bottlenecks.
+to reduce network bandwidth bottlenecks.
 
-There was a big caveat with this: The data users would download from
-knightmb https://bitcoin.stackexchange.com/a/113682/69518[wasn't
+There was a big caveat with this: all the data users would download from
+knightmb https://bitcoin.stackexchange.com/a/113682/69518[weren't
 verified by the Bitcoin software] at startup. The blkindex.dat file
 contained the UTXO set, and the software would accept any data therein
 as if it had already verified it. knightmb could have manipulated the
-data to give himself some bitcoins.
+data to give himself or anyone else some bitcoins.
 
 Again, people seemed to go along with this, and the reversal of the
-invalid block and its successors was successful. The miners started
+invalid block and its successors was successful. Miners started
 working on a new successor to block
 https://mempool.space/block/0000000000606865e679308edf079991764d88e8122ca9250aef5386962b6e84[74637]
-and according to the block's timestamp, a successor appeared on August
-15 at 23:53 UTC, about 6 hours after the issue was discovered. At
-08:10 on August 16 around block 74689, it seems the new chain had
-overtaken the old chain, and all non-upgraded nodes reorged to follow
-the new chain. This is the deepest reorg, 52 blocks, in Bitcoin's
-history.
+and, according to the block's timestamp, a successor appeared at 23:53
+UTC, about 6 hours after the issue was discovered. At 08:10 on August
+16 around block 74689, the new chain had overtaken the old
```

Comment:
Since the date August 15 was removed above, we probably should say "At 08:10 the day after, on August 16, around block 74689" so that the reader understands clearly how much time has passed.

Reply:
It sounds clearer, you're right